packages/atlas/lib/src/marker.dart

```diff
@@ -76,3 +79,34 @@ class Anchor {
   /// {@macro anchor}
   const Anchor({this.x = 0.5, this.y = 0.5});
 }
+
+/// Text labels for a [Marker] info window.
+class CallOut {
```

Comment:
Rather than CallOut, could we name this Annotation since Marker Annotation may be a better understood name.